redirects.yml

```diff
@@ -14,8 +14,8 @@ redirects:
   'solutions/observability/apps/application-performance-monitoring-apm.md': 'solutions/observability/apm/index.md'
   'solutions/observability/apps/get-started-with-apm.md': 'solutions/observability/apm/get-started.md'
   'solutions/observability/apps/get-started-apm-serverless.md': 'solutions/observability/apm/get-started.md'
-  'solutions/observability/apps/fleet-managed-apm-server.md': 'solutions/observability/apm/apm-server-fleet-managed.md'
```

Comment:
Are we sure we want to remove these existing redirects? Are already public an in use?